sdk-python/tests/copilotkit/langgraph_agent/agent.py

```
@@ -0,0 +1,146 @@
+"""
+A LangGraph implementation of the human-in-the-loop agent.
+"""
+
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)